test/unit/ngsi-communication-test.js

```diff
@@ -200,12 +206,12 @@ describe('Context Broker communication', function() {
             }
         };
 
-        it('should raise a controlled error', function(done) {
-            request(options, function(error, response, body) {
+        it('should raise a controlled error', function (done) {
+            request(options, function (error, response, body) {
                 should.not.exist(error);
                 response.statusCode.should.equal(404);
                 done();
             });
         });
     });
-});
+});
```

Comment:
```suggestion

        describe('When a sigfox measure fails updating Context Broker', function () {
            const options = {
                url: 'http://localhost:' + config.sigfox.port + '/update',
                method: 'GET',
                qs: {
                    id: 'sigApp1',
                    time: 1430909015,
                    statin: '0A5F',
                    lng: -4,
                    lat: 41,
                    data: '000000020000000000230c6f'
                }
            };
    
            it('should raise a controlled error', function (done) {
                request(options, function (error, response, body) {
                    should.not.exist(error);
                    response.statusCode.should.equal(200);
                    done();
                });
            });
        });
    });
});
```

Adding a failing test to cover the line was uncovered after fixing the tests as mentioned [here](https://github.com/telefonicaid/sigfox-iotagent/pull/111#issuecomment-919153077)